pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -352,11 +352,14 @@ func dryRunPreemption(ctx context.Context, fh framework.Handle,
 			if nvcSize > 0 && nvcSize+vcSize >= numCandidates {
 				cancel()
 			}
-		} else {
-			statusesLock.Lock()
-			nodeStatuses[nodeInfoCopy.Node().Name] = status
-			statusesLock.Unlock()
+			return
+		}
+		if status.IsSuccess() && len(pods) == 0 {
+			status = framework.AsStatus(fmt.Errorf("invalid victims on node %q returned by selectVictimsOnNode", nodeInfoCopy.Node().Name))
```

Comment:
right, be more explicit, like "expected at least one victim on node $q"

Reply:
Updated the error msg. as suggested.

Reply:
> right, be more explicit, like "expected at least one victim on node $q"

Updated the error msg. as suggested.